tracing-attributes/src/attr.rs

```diff
@@ -68,6 +70,17 @@ impl InstrumentArgs {
         }
     }
 
+    /// The location of the `tracing` crate.
+    ///
+    /// For backwards-compatibility, this must default to the plain `tracing` identifier with call site resolution.
```

Comment:
I'm not entirely sure if this should go into the doc comment, on second thought. A comment above the `quote!(tracing)` may be more apt.

I'd actually like to suggest adding one of those breaking change comments there: A default of `quote!(::tracing)` would avoid collisions with consumer-defined names just about entirely by default, but unfortunately it would also break the "hidden override" that's theoretically available right now.